### ISSUE TITLE
Set a starting balance and accounts

### ISSUE THREAD

**Issue by mastro-ale** [rafsoh/dimeApp#55]
### Prerequisites

- [X] I have searched for [existing issues](https://github.com/rarfell/dimeApp/issues) that already include this feature request, without success.

### Describe the Feature Request

I have been an avid user of expense tracking apps, from 2019 to the end the one I use the most is **Debit & Credit**. Like many other apps, I think that Dime also needs the management of multiple accounts and above all an initial amount that can be set.

### Describe the Use Case

To start using the app, the user is likely to start from their bank account which is already their piggy bank in itself. An app like this therefore cannot help but start from an amount set for an account that simulates the performance of one's bank account, otherwise everyone would always have to start with the import of a .csv which is a non-immediate use case and requires knowledge of spreadsheets not superfluous.

### Additional Information

I like the simplicity and modern design of the Dime app and I'd like to see some more simple features already existing in many other expense trackers, that would make the Dime app the best in the iOS ecosystem.

**Comment by EdgarDev17** [comment]
I have also thought that the feature of aggregating accounts is something important to keep better control of finances.

The app is very good, it is well worked, in fact I would also like the author to find a way to monetize it, so he can dedicate himself to it, offer a quality app and receive money in return.

**Comment by gauravlonari** [comment]
Yeah it will be very better if we have this support in Dime. All the other features are great, if this is added, it will be the best amongst all

**Comment by davidruano3** [comment]
Please Implement this! Also to do adjustments in the balance if at certain point it what the app says doesn’t match with reality. Those adjustments should not impact expenses or income

**Comment by Misnad** [comment]
This missing feature makes me want to use another expense tracker. This is a must for me.

**Comment by Misnad** [comment]
This missing feature makes me want to use another expense tracker. This is a must for me.
